unittests/resources/testlib/simple.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class simple_echo(rfm.RunOnlyRegressionTest, pin_prefix=True):
+    descr = 'Simple Echo build fixture'
+    executable = 'echo'
+    executable_opts = ['Hello']
+
+    @sanity_function
+    def assert_success(self):
+        return sn.assert_found(r'Hello', self.stdout)
```

Comment:
This can also be made very dummy and compact as it's sole purpose is to create dependency:

```suggestion
class dummy_fixture(rfm.RunOnlyRegressionTest):
    executable = 'echo'
    sanity_patterns = sn.assert_true(1)
```